jme3-core/src/main/java/com/jme3/renderer/opengl/GLRenderer.java

```diff
@@ -3268,9 +3270,8 @@ public void setMainFrameBufferSrgb(boolean enableSrgb) {
         setFrameBuffer(null);
 
         if (enableSrgb) {
-            if (!getBoolean(GLExt.GL_FRAMEBUFFER_SRGB_CAPABLE_EXT)) {
-                logger.warning("Driver claims that default framebuffer "
-                        + "is not sRGB capable. Enabling anyway.");
+            if (JmeSystem.getPlatform() != Platform.MacOSX64 && JmeSystem.getPlatform() != Platform.MacOSX_ARM64 && !getBoolean(GLExt.GL_FRAMEBUFFER_SRGB_CAPABLE_EXT)) {
```

Comment:
1. There ought to be a comment here explaining why `MacOSX64` and `MacOSX_ARM64` are exempt from the `getBoolean()` call.
2. Have you seen the 1280 error on both platforms, or just on `MacOSX_ARM64`?
3. If we need to exempt both platforms, then the code would be more readable if you saved the `getPlatform()` result to a local variable and tested that variable, rather than invoking `getPlaform()` twice.

Reply:
They are the only two architectures that are still supported by apple. And MacOS running on Apple M1 (that is the arm64 processor) can also run x86 64bit apps. 
I exempted only those two because the others are deprecated and i don't know if all the old systems running there support sRGB.
I don't agree with point 3

Reply:
1. I understand the reason for exempting ARM-based Macs (1280 errors) because I'm reading the code in the context of this PR. But a future developer reading the code without that context won't know about the 1280 errors. So I still think a comment should be added.
2. If you've only seen the 1280 error with `MacOSX_ARM64`, then why do you believe it's necessary to test for `MacOSX64`?

Reply:
1. Ok i'll add a comment
2. Because I believe this issue is with the OS and they run the same OS, in any case this only removes a warning, it won't impact with the engine functionality

Reply:
OK.